lib/src/data_frame/helpers/data_frame_to_string.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:ml_dataframe/ml_dataframe.dart';
```

Comment:
@tadeohepperle could you please replace the index file import with the direct one? I see, that you use `DataFrame` class, so you can do the following import:

```dart
import 'package:ml_dataframe/src/data_frame/data_frame.dart';
```

instead of 

```dart
import 'package:ml_dataframe/ml_dataframe.dart';
```

Index files such as `package:ml_dataframe/ml_dataframe.dart` are only for library external usage, inside the library, it is better to use  imports from the `src` directory